docs/source/getting-started/tracking-server-overview/index.rst

```diff
@@ -0,0 +1,327 @@
+
+.. code-block:: python
+
+  mlflow.set_experiment("/check-localhost-connection")
+
+  with mlflow.start_run():
+      mlflow.log_metric("foo", 1)
+      mlflow.log_metric("bar", 2)
+
+
+Putting it together you can copy the following code to your editor and save it as `test_mlflow_localhost.py`:
+
+.. code-block:: python
+
+  import mlflow
+
+  mlflow.set_tracking_uri("http://localhost:5000")
+  mlflow.set_experiment("check-localhost-connection")
+
+  with mlflow.start_run():
+      mlflow.log_metric("foo", 1)
+      mlflow.log_metric("bar", 2)
+
+
+Then execute it by:
+
+.. code-block:: bash
+
+  $ python test_mlflow_localhost.py
+
+
+View Experiment on Your MLflow Server
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Now let's view your experiment on the local server. Open the URL in your browser, which is `http://localhost:5000` 
+in our case. In the UI, inside the left sidebar you should see the experiment with name 
+`"check-localhost-connection"`, clicking on it should bring you to the experiment view similar as below. 
+
+.. image:: ../../_static/images/quickstart/tracking-server-overview/mlflow-localhost-experiment-view.png
+    :width: 800px
+    :align: center
+    :alt: Experiment view of OSS MLflow server
+
+Clicking on the run (`"clumsy-steed-426"`) will bring you to the run view, similar as below.
+
+.. image:: ../../_static/images/quickstart/tracking-server-overview/mlflow-localhost-run-view.png
+    :width: 800px
+    :align: center
+    :alt: Run view of OSS MLflow server
+
+Conclusion
+^^^^^^^^^^^
+
+That's all about how to start your own MLflow server and view your experiments. Please see the pros and cons
+of this method below:
+
+* **Pros**
+  
+  * You have full control of your data and model, which is good for privacy concerns.
+  * No subscription is required.
+  * Unlimited quota of experiments/runs.
+  * You can even customize your UI by forking the MLflow repo and modify the UI code.
+
+* **Cons**
+  
+  * Requires manual setup and maintenance.
+  * Team collaboration is harder than using a hosted tracking server, e.g., Databricks CE.
+  * Not suitable for cloud-based notebook, e.g., Google Colab.
+  * Requires extra port forwarding if you deploy your server on cloud VM.
+  * No serving support.
+
+
+Method 2: Use Free Hosted Tracking Server (Databricks Community Edition)
+------------------------------------------------------------------------
+
+**Notice**: This part of guide can be directly executed in cloud-based notebook, e.g., Google Colab or 
+Databricks Notebook.
+
+`Databricks Community Edition (CE) <https://community.cloud.databricks.com/>`_ is the free version of 
+cloud-based big data platform Databricks. Databricks CE users can access a micro-cluster as well as 
+a cluster manager and notebook environment. All users can share their notebooks and host them free of 
+charge with Databricks. You can use Databricks CE to store and view your MLflow experiments without 
+being charged.
```

Comment:
We might want to clean this wording up a bit. It's great for getting started and learning the basics of Databricks / MLflow, but it's definitely not intended for long-term use (I think there's a purge that happens, so there's not even a guarantee of data consistency and longevity; we definitely don't want to make it seem like people can use CE as a solution in and of itself. It's just for teaching / learning purposes only.

Reply:
CE is actually okay for long-term use if users just want to track, which covers a decent portion of users. I did some search and the experiments won't be deleted automatically. My experiments from 3 months ago are still accessible.

From next Q2 or so, we can refactor this part to rely on Lighthouse.